cypress/e2e/calendar/edit-hours-of-operation.cy.js

```diff
@@ -0,0 +1,129 @@
+import {
+  MultiColumnListCell, MultiColumnListRow, TimeDropdown, Pane, Button, IconButton, TextField, Accordion,
+  including, Link, matching
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+// delete all but the first opening time
+testCalendar.normalHours = [
+  {
+    startDay: 'Monday',
+    startTime: '09:00',
+    endDay: 'Thursday',
+    endTime: '02:00'
+  }
+];
+
+const editHoursOfOperationData = calendarFixtures.data.editHoursOfOperation;
+const editHoursOfOperationExpectedUIValues = calendarFixtures.expectedUIValues.editHoursOfOperation;
+
+
+describe('Edit existing hours of operation for service point', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+      cy.openCalendarSettings();
+    });
+
+    cy.openCalendarSettings();
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('edits hours of operation for an open day', () => {
```

Comment:
it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {